### PR TITLE
ferrari:update

### PR DESCRIPTION
**Pull request by hashroot**

### CODE REVIEW

File: orion.mk [hunk 1]
```diff
@@ -26,11 +26,17 @@ PRODUCT_MANUFACTURER := Xiaomi
...
     TARGET_DEVICE="ferrari" \
     PRODUCT_NAME="ferrari" \
     BUILD_FINGERPRINT="Xiaomi/ferrari/ferrari:6.0.1/MMB29M:userdebug/test-keys" \
     PRIVATE_BUILD_DESC="ferrari-userdebug 6.0.1 MMB29M test-keys"
+
+# BootAnimation Path
+TARGET_SCREEN_HEIGHT := 1280
+TARGET_SCREEN_WIDTH := 720
+PRODUCT_COPY_FILES += \
+     vendor/orion/config/media/bootanimation.zip:system/media/bootanimation.zip
```
Comment (akhilnarang) [inline orion.mk:42]: o.O

**Comment by hashroot** [comment]
wut?

File: orion.mk (hunk 1 above)
Comment (hashroot) [inline orion.mk:42]: wut?

**Comment by owlseye** [comment]
https://github.com/TeamOrion/vendor_orion/blob/6.0.1/config/common.mk#L152

**Comment by hashroot** [comment]
done thanks can u pls trigger the build ?

**Comment by owlseye** [comment]
Lol, where?

**Comment by hashroot** [comment]
u got server access ?

**Comment by owlseye** [comment]
@minidude2012 
Server is gone, right?

**Comment by akhilnarang** [comment]
Server is there for a day moar I guess :P

File: orion.mk (hunk 1 above)
Comment (akhilnarang) [inline orion.mk:42]: This'll give error

Comment (akhilnarang) [inline orion.mk:42]: Why is that line even there :|

**Comment by akhilnarang** [comment]
Deps aren't proper

**Comment by owlseye** [comment]
And why you add all build combos to lunch selection?

**Comment by hashroot** [comment]
https://github.com/TeamOrion-Devices/device_yu_lettuce/blob/d6ce8f824ee54cc2f46ba82f1fff6e83e83965c0/vendorsetup.sh

**Comment by akhilnarang** [comment]
Mehh vendorsetup dosent matter😂

**Pull request merged**